libass/ass.h

```diff
@@ -238,6 +238,14 @@ typedef enum {
      */
     ASS_FEATURE_BIDI_BRACKETS = 1 << 1,
 
+    /**
+     * If no explicit ("\n") or implicit (" ") wrap point is found, wrap at the
+     * last seen character when doing smart wrap.
+     *
+     * Has no effect if wrap_mode is 2.
+     */
+    ASS_FEATURE_ALWAYS_WRAP = 1 << 2,
```

Comment:
I think a differnt name would be better. Eg  `WRAP_BREAKWORDS` or a bit longer but also clearer `WRAP_FORCE_ON_OVERFLOW`.